.github/readme.md

```diff
@@ -47,17 +47,17 @@ First Install The dependencies for Debian-like distributions.
 
 ``sudo mount -t exfat -o uid=`id -u`,gid=`id -g` ./ps4-hdd.exfat ./ps4-fs``
 
-## How to Run samples and games ( one day )
+## How to run samples and games ( one day )
    
 See usage message of `rpcsx-os` (`-h` argument), or join the [Discord](https://discord.gg/t6dzA4wUdG) for help.
 
-You can run emulator with Samples using this command:
+You can run the emulator with some samples using this command:
    
 `rm -f /dev/shm/rpcsx-* && ./rpcsx-os --mount  "<path to fw>/system" "/system" --mount "<path to 'game' root>" /app0 /app0/some-test-sample.elf [<args for test elf>...]`
 
 ## Creating a log
 
-You can use this flag if you get sigfault for debugging.
+You can use this flag if you got a sigfault for debugging.
```

Comment:
Why only grammar, and leave spelling?
`segfault`
"got a" could be "encounter", "experience", or "receive".

Reply:
will fix it